config/json-schema/rpc-invited-organisations-bulk-delete-schema.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "bulk.delete.invited.organisations",
```

In the other cases the the id starts with the entity/resourceType and using camel-case, I think we should use the usual concept here.
```suggestion
  "$id": "invitedOrganisations.bulk.delete",
```